client/src/css/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
```

Comment:
have you considered using SCSS instead? It has all the functionality of CSS with a few other cool features, for example being able to use variables.

Reply:
Yeah, I would like to use SCSS - I'll add a ZenHub ticket for it. It didn't seem like it was necessary for this case, so I omitted it from these changes.

Reply:
Sounds good, we can add it when we start needing to style the frontend.

Reply:
Ticket added to backlog!